### DISCUSSION TITLE
How to install on QNAP

### DISCUSSION THREAD

**Discussion by ptfuzi** [transmission/transmission#6770] [Q&A]
### What is the issue?

Can’t install on QNAP x86 nas, how to build installer file?

### Which application of Transmission?

Qt app on Linux, BSD, etc.

**Comment by killemov** [comment]
I replaced the QNAP firmware (armel) with Debian a long time ago and never looked back. I know there used to be a Transmission QPKG, but I don't know if that is still being maintained.

Your question is only specific for QNAP, so https://www.google.com/search?q=how+to+build+a+qnap+package ?

**Reply by SNBogdanov** [reply]
Install OPKG and then use it to install transmission
opkg list |grep transmission
transmission-cli - 4.0.3-4 - CLI utilities for transmission.
transmission-daemon - 4.0.3-4 - Transmission is a simple BitTorrent client. It features a very simple, intuitive interface on top on an efficient, cross-platform back-end. This package contains the daemon itself.
transmission-remote - 4.0.3-4 - CLI remote interface for transmission.
transmission-web - 4.0.3-4 - Webinterface resources for transmission.
transmission-web-control - 2021-09-25-4b2e1858-3 - Transmission Web Control is a custom Web UI for Transmission.
